### PR TITLE
Rename Relic's custom CFLAGS to COMP_FLAGS. Addresses issue #267.

### PR DESCRIPTION
**Pull request by AmineKhaldi**

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -4,14 +4,15 @@ set (CMAKE_CXX_STANDARD 17)
...
+  # This is currently anchored to upstream b7b2266a0e4ee6f628f61d3ab638f524a18b52f1
+  set(RELIC_GIT_TAG "d20d97c046da2c1689916569e9895b48a4693624")
 endif ()
 
 message(STATUS "Relic will be built from: ${RELIC_GIT_TAG}")
 
 FetchContent_Declare(
   relic
-  GIT_REPOSITORY https://github.com/relic-toolkit/relic.git
+  GIT_REPOSITORY https://github.com/AmineKhaldi/relic.git
```
Comment (hoffmang9) [inline src/CMakeLists.txt:15]: Are you planning to stay with your forked version or is this a development artifact?

Comment (AmineKhaldi) [reply src/CMakeLists.txt:15]: It's intentional yes, as I was advised to fork while we try to upstream eventually. I sent this one upstream already (https://github.com/relic-toolkit/relic/pull/207) but more will come.

Comment (hoffmang9) [reply src/CMakeLists.txt:15]: Just wanted to confirm

**Review by hoffmang9 — APPROVED** [review]
aok

**Pull request merged**